docs/modules/ROOT/pages/migration/servlet/config.adoc

```diff
@@ -409,7 +409,7 @@ import static org.springframework.security.web.util.matcher.AntPathRequestMatche
 @Bean
 public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
     http
-        .securityMatcher(antMatcher("/api/**"), antMatcher("/app/**"))
+        .securityMatchers().requestMatchers(antMatcher("/api/**"), antMatcher("/app/**"))
```

Comment:
We recommend using the lambda customizer to configure the DSL.

```suggestion
        .securityMatchers((matchers) -> matchers
            .requestMatchers(antMatcher("/api/**"), antMatcher("/app/**"))
        )
```

Reply:
ah good catch, otherwise I probably also would have needed to add a `.and()` after the call to `requestMatchers()` :)